react-native-xmpp.podspec

```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+  s.name         = "react-native-xmpp"
+  s.version      = "0.2.1"
+  s.license      = "MIT"
+  s.homepage     = "https://github.com/aksonov/react-native-xmpp"
+  s.authors      = { 'Marc Shilling' => 'marcshilling@gmail.com' }
```

Comment:
I don't understand why we need cocoa pods at all. rnpm works fine and it is expected way to work with Native components and React Native.